Kite SDK/PSPrintSDK/OLAnalytics.m

```diff
@@ -123,10 +123,11 @@ + (void)trackReviewScreenViewed:(NSString *)productName{
     [OLAnalytics sendToMixPanelWithDictionary:dict];
 }
 
-+ (void)trackShippingScreenViewedForOrder:(OLPrintOrder *)printOrder{
++ (void)trackShippingScreenViewedForOrder:(OLPrintOrder *)printOrder variant:(NSString *)variant{
     NSDictionary *dict = [OLAnalytics defaultDictionaryForEventName:@"Shipping Screen Viewed"];
     NSMutableDictionary *p = [self propertiesForPrintOrder:printOrder];
     [dict[@"properties"] addEntriesFromDictionary:p];
+    [dict[@"properties"] addEntriesFromDictionary:@{@"Shipping Screen Variant" : variant}];
```

Comment:
This is clearly OK but am I missing something, how come you can mutate an NSDictionary rather than NSMutableDictionary?